src/clj/rems/workflow/dynamic.clj

```diff
@@ -189,6 +189,13 @@
              :application/request-id last-request-for-actor
              :application/comment (:comment cmd)}))))
 
+(defmethod command-handler :application.command/add-licenses
+  [cmd _application injections]
+  (or (must-not-be-empty cmd :licenses)
+      (ok {:event/type :application.event/licenses-added
+           :application/licenses (set (:licenses cmd))
```

Comment:
should these be in `{:license/id ...}` format since the key is `:application/licenses`?

Reply:
Good idea, I did this refactoring.